ee/tabby-webserver/Cargo.toml

```diff
@@ -11,6 +11,7 @@ async-trait.workspace = true
 axum = { workspace = true, features = ["ws"] }
 bincode = "1.3.3"
 chrono = "0.4"
+filenamify = "0.1.0"
```

Comment:
I suggest have a unified place to convert a git url to name - current implementation let `tabby-scheduler` and `tabby-webserver` each rolls their own implementation